wrapanapi/systems/ec2.py

```diff
@@ -886,13 +885,15 @@ def delete_sqs_queue(self, queue_url):
     def get_all_unused_loadbalancers(self):
         return [
             loadbalancer for loadbalancer
-            in self.elb_connection.get_all_load_balancers()
-            if not loadbalancer.instances]
+            in self.elb_connection.describe_load_balancers().get("LoadBalancerDescriptions")
+            if not loadbalancer.get("Instances")]
 
     def delete_loadbalancer(self, loadbalancer):
-        self.logger.info(" Deleting Elastic Load Balancer '%s'", loadbalancer.name)
+        self.logger.info(" Deleting Elastic Load Balancer '%s'",
```

Comment:
API change here, where we were expecting a loadbalancer object before, its now a dictionary?  Needs to be in docblock what is expected, and I don't think it makes sense to take a dictionary here, it should take the loadbalancer name directly.

Reply:
Sure, but boto3 returns dictionary instead of objects as it was in boto. So I have to return dict or that name.